CMakeLists.txt

```diff
@@ -63,66 +63,66 @@ if(OTIO_PYTHON_INSTALL)
         set(OTIO_RESOLVED_PYTHON_INSTALL_DIR "${Python_SITEARCH}")
         set(OTIO_RESOLVED_CXX_INSTALL_DIR "${Python_SITEARCH}/opentimelineio/cxx-libs")
         set(OTIO_RESOLVED_CXX_DYLIB_INSTALL_DIR "${Python_SITEARCH}/opentimelineio")
-        message(INFO, "OTIO Defaulting both Python and C++ install to ${OTIO_RESOLVED_PYTHON_INSTALL_DIR}")
+        message(STATUS "OTIO Defaulting both Python and C++ install to ${OTIO_RESOLVED_PYTHON_INSTALL_DIR}")
     else()
         # either python_install or install_prefix have been set
         if(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
-            # OTIO_PYTHON_INSTALL_DIR was set, so install everything into the python package
+            # OTIO_PYTHON_INSTALL_DIR was set, so install everything into the Python package
             set(OTIO_RESOLVED_PYTHON_INSTALL_DIR "${OTIO_PYTHON_INSTALL_DIR}")
             set(OTIO_RESOLVED_CXX_INSTALL_DIR "${OTIO_PYTHON_INSTALL_DIR}/opentimelineio/lib")
             set(OTIO_RESOLVED_CXX_DYLIB_INSTALL_DIR "${OTIO_PYTHON_INSTALL_DIR}/opentimelineio")
-            message(INFO, "OTIO Defaulting C++ install to ${OTIO_RESOLVED_CXX_INSTALL_DIR}")
+            message(STATUS "OTIO Defaulting C++ install to ${OTIO_RESOLVED_CXX_INSTALL_DIR}")
         else()
-            # CMAKE_INSTALL_PREFIX was set, so install the python components there
+            # CMAKE_INSTALL_PREFIX was set, so install the Python components there
             set(OTIO_RESOLVED_PYTHON_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/python")
             set(OTIO_RESOLVED_CXX_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}")
 
             # In order to not require setting $PYTHONPATH to point at the .so,
-            # the shared libraries are installed into the python library 
+            # the shared libraries are installed into the python library
             # location.
             set(OTIO_RESOLVED_CXX_DYLIB_INSTALL_DIR "${OTIO_PYTHON_INSTALL_DIR}/opentimelineio")
-            message(INFO, "OTIO Defaulting Python install to ${OTIO_RESOLVED_PYTHON_INSTALL_DIR}")
-            message(INFO, "Note: C++ linkable-shared libraries can be found: ${OTIO_PYTHON_INSTALL_DIR}/opentimelineio")
+            message(STATUS "OTIO Defaulting Python install to ${OTIO_RESOLVED_PYTHON_INSTALL_DIR}")
+            message(STATUS "Note: C++ linkable shared libraries can be found at: ${OTIO_PYTHON_INSTALL_DIR}/opentimelineio")
```

Comment:
This could be the solution:

```diff
diff --git CMakeLists.txt CMakeLists.txt
index dba361e..e1ebaff 100644
--- CMakeLists.txt
+++ CMakeLists.txt
@@ -50,8 +50,8 @@ if(OTIO_PYTHON_INSTALL)
     #   Python: ${OTIO_PYTHON_INSTALL_DIR}/opentimelineio
     #   C++:    ${OTIO_PYTHON_INSTALL_DIR}/opentimelineio/lib
     # if only CMAKE_INSTALL_PREFIX has been set,
-    #   Python: ${CMAKE_INSTALL_PREFIX}/opentimelineio/python
-    #   C++:    ${CMAKE_INSTALL_PREFIX}/opentimelineio
+    #   Python: ${CMAKE_INSTALL_PREFIX}/python/opentimelineio
+    #   C++:    ${CMAKE_INSTALL_PREFIX}/python/opentimelineio
     #
     # In a Python install, the dylibs/dlls need to be installed where __init__.py
     # can find them, rather as part of the C++ SDK package; so the variable
@@ -76,13 +76,13 @@ if(OTIO_PYTHON_INSTALL)
             # CMAKE_INSTALL_PREFIX was set, so install the Python components there
             set(OTIO_RESOLVED_PYTHON_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/python")
             set(OTIO_RESOLVED_CXX_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}")
+            message(STATUS "OTIO Defaulting Python install to ${OTIO_RESOLVED_PYTHON_INSTALL_DIR}")

             # In order to not require setting $PYTHONPATH to point at the .so,
             # the shared libraries are installed into the python library
             # location.
-            set(OTIO_RESOLVED_CXX_DYLIB_INSTALL_DIR "${OTIO_PYTHON_INSTALL_DIR}/opentimelineio")
-            message(STATUS "OTIO Defaulting Python install to ${OTIO_RESOLVED_PYTHON_INSTALL_DIR}")
-            message(STATUS "Note: C++ linkable shared libraries can be found at: ${OTIO_PYTHON_INSTALL_DIR}/opentimelineio")
+            set(OTIO_RESOLVED_CXX_DYLIB_INSTALL_DIR "${OTIO_RESOLVED_PYTHON_INSTALL_DIR}/opentimelineio")
+            message(STATUS "Note: C++ linkable shared libraries can be found at: ${OTIO_RESOLVED_CXX_DYLIB_INSTALL_DIR}")
         endif()
     endif()
 else()
```